level2/loadbalancers.tf

```diff
@@ -0,0 +1,82 @@
+resource "aws_lb" "lb_app" {
+  name               = "lb-tf-application"
+  internal           = false
+  load_balancer_type = "application"
+  security_groups    = [aws_security_group.lb_sec_gpe.id]
+  subnets            = data.terraform_remote_state.level1.outputs.public_subnet
+}
+
+resource "aws_security_group" "lb_sec_gpe" {
+
+  name   = "${var.env_code}-loadbalacer"
+  vpc_id = data.terraform_remote_state.level1.outputs.vpc_id
+
+  ingress {
+    from_port   = 80
+    to_port     = 80
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+  ingress {
```

Comment:
you have the same ingress rule of port 80. could you please remove one of them as they are the same thing

Reply:
could you please remove one ingress rule of port 80 as they exist twice